src/7.1_HW/Department.ts

```diff
@@ -0,0 +1,37 @@
+import {Employee, Status} from "./Employee";
+import {PreHiredEmployee} from "./PreHiredEmployee";
+
+export class Department {
+    name: string;
+    domain: string;
+    employees: Employee[];
+    budget: Budget;
+
+    constructor(name: string, domain: string, employees: Employee[], budget: Budget) {
+        this.name = name;
+        this.domain = domain;
+        this.employees = employees;
+        this.budget = budget;
+    }
+
+    calculateBalance(): number {
+        return this.budget.debit - this.budget.credit;
+    }
+
+    addEmployee(employee: Employee) {
```

Comment:
Please, do not forget to add type annotation for return function type explicitly.